doc/source/spectra.rst

```diff
@@ -317,6 +317,66 @@ Plotting
 
 See :ref:`Plotting <plotting>`
 
+Kinematic constraints
+---------------------
+
+Inelastic neutron-scattering (INS) experiments are often performed using
+time-of-flight information to sample a wide range of energy transfer.
+In "direct geometry" the incident energy is fixed (e.g. by a Fermi
+chopper) while in "indirect geometry" the measurement energy is fixed
+(e.g. by scattering from an "analyser" crystal).
+Conservation laws allow the overall energy and momentum
+transfer to be determined for a given scattering angle --- but this also
+means that the accessible :math:`(q, \omega)` range is restricted by
+these instrument parameters.
+
+The function :py:func:`apply_kinematic_constraints
+<euphonic.spectra.apply_kinematic_constraints>` applies these limits
+to a :ref:`Spectrum2D` object with appropriate dimensions (i.e. the x-
+and y-axes represent :math:`|q|` and :math:`\omega` respectively).
```

Comment:
```suggestion
to a powder averaged :ref:`Spectrum2D` object with appropriate dimensions
(i.e. the x-and y-axes represent :math:`|q|` and :math:`\omega` respectively).
```

Reply:
Re-read it in context and you had that bang on. Cheers.